graphdatascience/graph/graph_entity_ops_runner.py

```diff
@@ -36,7 +36,7 @@ def _handle_properties(
         return self._query_runner.run_query(query, params)
 
 
-class GraphPropertyRunner(GraphEntityOpsBaseRunner):
+class GraphElementPropertyRunner(GraphEntityOpsBaseRunner):
```

Comment:
is element and entity the same thing here?
its a small detail but maybe we could use the same term (if indeed it refers to the same thing)